rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -2093,13 +2129,17 @@ void registerFlyoverBtcTransaction_failed_when_tx_with_witness_surpassing_lockin
         FeePerKbSupport feePerKbSupport = mock(FeePerKbSupport.class);
         when(feePerKbSupport.getFeePerKb()).thenReturn(Coin.MILLICOIN);
 
+        Coin lockingCapValue = Coin.COIN;
+        when(lockingCapSupport.getLockingCap()).thenReturn(Optional.of(lockingCapValue));
+
         FederationSupport federationSupport = federationSupportBuilder
             .withFederationConstants(bridgeConstants.getFederationConstants())
             .withFederationStorageProvider(federationStorageProvider)
             .withActivations(activations)
             .build();
 
         BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withLockingCapSupport(lockingCapSupport)
```

Comment:
same here, move it closer to `[here](https://github.com/rsksmart/rskj/pull/2663/files#diff-e595294bfc4b99fd081ffaf7c754cdf9b64bd6b9c8494f71d6ddaf957bf8045bR2150)`

Reply:
Done